lib/sqlalchemy/engine/result.py

```diff
@@ -1129,12 +1142,12 @@ def columns(self, *col_expressions: _KeyIndexType) -> Self:
         return self._column_slices(col_expressions)
 
     @overload
-    def scalars(self: Result[Tuple[_T]]) -> ScalarResult[_T]:
+    def scalars(self: Result[_T, Unpack[_RawRowType]]) -> ScalarResult[_T]:
         ...
 
     @overload
     def scalars(
-        self: Result[Tuple[_T]], index: Literal[0]
+        self: Result[_T, Unpack[_RawRowType]], index: Literal[0]
```

Comment:
**Michael Bayer** (zzzeek) wrote:

_RawRowType should be TupleAny

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986